presidio-analyzer/Dockerfile

```diff
@@ -17,14 +17,14 @@ COPY ${NLP_CONF_FILE} /usr/bin/${NAME}/${NLP_CONF_FILE}
 
 WORKDIR /usr/bin/${NAME}
 
+COPY ./pyproject.toml /usr/bin/${NAME}/
 COPY ./Pipfile* /usr/bin/${NAME}/
 
 # Install essential build tools
 RUN apt-get update \
   && apt-get install -y build-essential
 
-RUN pip install pipenv \
-  && pipenv install --deploy
+RUN pip install pipenv && pipenv install pipenv install -e '.[server,stanza,azure-ai-language]'
```

Comment:
Should the other Dockerfiles be updated too?

Reply:
Yes, I was POCing with analyzer